src/js/controller/LayersListController.js

```diff
@@ -24,6 +24,15 @@
 
     $.subscribe(Events.PISKEL_RESET, this.renderLayerList_.bind(this));
     $.subscribe(Events.USER_SETTINGS_CHANGED, $.proxy(this.onUserSettingsChange_, this));
+
+    if (!Constants.ENABLE_MULTIPLE_LAYERS) {
```

Comment:
I would move this check to the top of the function and bail out after hiding the element (no need to attach events for this controller if it doesn't get displayed:
```javascript
ns.LayersListController.prototype.init = function () {
  this.rootEl = document.querySelector('.layers-list-container');
  if (!Constants.ENABLE_MULTIPLE_LAYERS) {
    this.rootEl.style.display = 'none';
    return;
  }
  // ... attach events etc...
};